snow/engine/snowman/bootstrap/bootstrapper.go

```diff
@@ -113,22 +126,34 @@ func New(config Config, onFinished func(ctx context.Context, lastReqID uint32) e
 		ChitsHandler:                common.NewNoOpChitsHandler(config.Ctx.Log),
 		AppHandler:                  config.VM,
 
+		minority: bootstrapper.Noop,
+		majority: bootstrapper.Noop,
+
 		Fetcher: common.Fetcher{
 			OnFinished: onFinished,
 		},
-		executedStateTransitions: math.MaxInt32,
-	}
-
-	config.Bootstrapable = b
-	b.Bootstrapper = common.NewCommonBootstrapper(config.Config)
-
-	return b, nil
+		executedStateTransitions: math.MaxInt,
+	}, err
 }
 
 func (b *Bootstrapper) Context() *snow.ConsensusContext {
 	return b.Ctx
 }
 
+func (b *Bootstrapper) GetVM() common.VM {
+	return b.VM
+}
```

Comment:
nit: outside the scope of this PR, but IIRC this getter was only needed to support some testing we don't have anymore (kurtosis). We could further trim down the interface and drop this getter if that is the case

Reply:
Addressed in #2374